docs/app/pages/components/sections/keyboard/hotkeys/hotkeys.component.ts

```diff
@@ -0,0 +1,123 @@
+            null, // require.context('!!prismjs-loader?lang=css!./snippets/', false, /\.css$/),
+            null, // require.context('!!prismjs-loader?lang=javascript!./snippets/', false, /\.js$/),
+            null, // require.context('!!prismjs-loader?lang=typescript!./snippets/', false, /\.ts$/),
+            require.context('./snippets/', false, /\.(html|less|js|ts)$/)
+        );
+
+        this.nativeElement = <HTMLElement> this.element.nativeElement;
+    }
+
+    hotkeys (event: KeyboardEvent) {
+        if (event.keyCode === 81) {
+            this.qKeypress();
+        }
+        if (event.keyCode === 87) {
+            this.wKeypress();
+        }
+    }
+
+    qKeypress() {
+        if (!this.qButtons) {
+            this.qButtons = <HTMLElement> this.nativeElement.children[0].children[1];
+        }
+        let focusedElement = <HTMLElement> this.nativeElement.children[0];
+        switch (this.qSelected) {
+            case 0:
+                focusedElement.focus();
+                this.qSelected = 1;
+                break;
+            case 1:
+                focusedElement = <HTMLElement> this.qButtons.children[0];
+                focusedElement.focus();
+                this.qSelected = 2;
+                break;
+            case 2:
+                focusedElement = <HTMLElement> this.qButtons.children[1];
+                focusedElement.focus();
+                this.qSelected = 3;
+                break;
+            case 3:
+                focusedElement = <HTMLElement> this.qButtons.children[2];
+                focusedElement.focus();
+                this.qSelected = 0;
+        }
+    }
+
+    updateQ(event: KeyboardEvent, text: string) {
+        if (event.keyCode === 32 || event.keyCode === 13) {
+            this.qText = text;
+        }
+    }
+
+    updateQClick(text: string) {
+        this.qText = text;
+    }
+
+    wKeypress() {
+        if (!this.wButtons) {
+            this.wButtons = <HTMLElement> this.nativeElement.children[1].children[1];
+        }
+        let focusedElement = <HTMLElement> this.nativeElement.children[1];
+        switch (this.wSelected) {
+            case 0:
+                focusedElement.focus();
+                this.wSelected = 1;
+                break;
+            case 1:
+                focusedElement = <HTMLElement> this.wButtons.children[0];
+                focusedElement.focus();
+                this.wSelected = 2;
+                break;
+            case 2:
+                focusedElement = <HTMLElement> this.wButtons.children[1];
+                focusedElement.focus();
+                this.wSelected = 3;
+                break;
+            case 3:
+                focusedElement = <HTMLElement> this.wButtons.children[2];
+                focusedElement.focus();
+                this.wSelected = 0;
+        }
+    }
+
+    updateW(text: string) {
+        this.wText = text;
+    }
+
+    public plunk: IPlunk = {
```

Comment:
Variable declarations should be placed above the constructor, only methods should be below